.github/release-drafter.yml

```diff
@@ -0,0 +1,30 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '💥 Breaking Changes'
+    labels:
+      - breaking
+  - title: '✨ Enhancements'
+    labels:
+      - feature
+      - enhancement
+  - title: '🐛 Bug Fixes'
+    labels:
+      - fix
+      - bugfix
+      - bug
+  - title: '🏗  Maintenance'
+    labels:
+      - chore
+      - dependencies
+change-title-escapes: '\<*_&#@'
+version-resolver:
+  major:
+    labels:
+      - breaking
+  minor:
+    labels:
+      - feature
+      - enhancement
+  default: patch
+template: '$CHANGES'
```

Comment:
Similar to the npm tag, we can add `prerelease: true` here to mark draft releases automatically as prereleases